### DISCUSSION TITLE
Concatenate videos in different formats

### DISCUSSION THREAD

**Discussion by DoctorMG** [mifi/lossless-cut#1497] [Q&A] [answered]
I have to concatenate videos in different formats. They shall run in an h5p training software and media need to be in some specified formats only.
So I try to bring a given video piece in the wanted format (mp4, h264, aac) - no big thing: I look for mismatching parameters and adapt them in ffmpeg.
On one mismatch I am stuck: the compatibility check shows: 
**Track 1 mismatch: level 32 != 42**

I have no idea, what _level_ this might be. Some audio normal, some video brightness, something with speed??? It seems there are plenty of levels. 
Does anybody know what level is meant here and maybe what ffmpeg switch would work?
Thanks

**Comment by mifi** [comment, marked as answer]
https://blog.mediacoderhq.com/h264-profiles-and-levels/

**Reply by Mrnofish** [reply]
I hope it's okay to piggy back on this question.

LosslessCut is warning that some files I am merging have different levels (3.0 and 3.1).

However, if I press merge, the operation goes ahead and the result appears to work fine in MPC-HC. Should I avoid merging without re-encoding? Any risks?

**Reply by mifi** [reply]
> I hope it's okay to piggy back on this question.

of course!

> However, if I press merge, the operation goes ahead and the result appears to work fine in MPC-HC. Should I avoid merging without re-encoding? Any risks?

hard to say unless consulting with an h264 codec expert. But if it seems to be working in the players you intend to use, i'd say why not?
